docs/src/content/docs/pt-br/guides/components.mdx

```diff
@@ -169,13 +218,200 @@ import { LinkCard } from '@astrojs/starlight/components';
 />
 
 <CardGrid>
-	<LinkCard
-		title="Escrevendo Markdown"
-		href="/pt-br/guides/authoring-content/"
-	/>
+	<LinkCard title="Escrevendo Markdown" href="/pt-br/guides/authoring-content/" />
 	<LinkCard title="Componentes" href="/pt-br/guides/components/" />
 </CardGrid>
 
+### Asides
+
+Asides (também conhecidos como "advertências" ou "frases de destaque") são úteis para exibir informações secundárias ao lado do conteúdo principal de uma página.
+
+Um `<Aside>` pode ter um `type` opcional de `note` (o padrão, nota), `tip` (dica), `caution` (cuidado) ou `danger` (perigo). A definição de um atributo `title` substitui o título padrão do aside.
+
+````mdx
+# src/content/docs/exemplo.mdx
+
+import { Aside } from '@astrojs/starlight/components';
+
+<Aside>Um aside padrão sem um título personalizado.</Aside>
+
+<Aside type="caution" title="Cuidado!">
+	Um aside de aviso *com* um título personalizado.
+</Aside>
+
+<Aside type="tip">
+
+Outros conteúdos também são compatíveis com asides.
+
+```js
+// Um trecho de código, por exemplo.
+```
+
+</Aside>
+
+<Aside type="danger">Não forneça sua senha a ninguém.</Aside>
+````
+
+O código acima gera o seguinte na página:
+
+import { Aside } from '@astrojs/starlight/components';
+
+<Aside>Um aside padrão sem um título personalizado.</Aside>
+
+<Aside type="caution" title="Cuidado!">
+	Um aside de aviso *com* um título personalizado.
+</Aside>
+
+<Aside type="tip">
+
+Outros conteúdos também são compatíveis com asides.
+
+```js
+// Um trecho de código, por exemplo.
+```
+
+</Aside>
+
+<Aside type="danger">Não forneça sua senha a ninguém.</Aside>
+
+O Starlight também fornece uma sintaxe personalizada para renderizar asides em Markdown e MDX como uma alternativa ao componente `<Aside>`.
+Consulte o guia ["Escrevendo Conteúdo em Markdown"](/pt-br/guides/authoring-content/#asides) para obter detalhes sobre a sintaxe personalizada.
+
+### Código
+
+Use o componente `<Code>` para renderizar código com syntax highlighting quando não for possível usar um [bloco de código Markdown](/pt-br/guides/authoring-content/#blocos-de-código), por exemplo, para renderizar dados provenientes de fontes externas, como arquivos, bancos de dados ou APIs.
+
+Consulte a [documentação do "Componente de código" do Expressive Code](https://expressive-code.com/key-features/code-component/) para obter detalhes completos sobre as propriedades suportadas pelo `<Code>`.
+
+```mdx
+# src/content/docs/exemplo.mdx
+
+import { Code } from '@astrojs/starlight/components';
+
+export const codigoExemplo = `console.log('Isso pode vir de um arquivo ou CMS!');`;
+export const nomeArquivo = 'exemplo.js';
+export const destaques = ['arquivo', 'CMS'];
```

Comment:
I personally always code in English and I believe is the most common practice in companies even here in Brazil, at least on all the big ones I know. With that said, I was following what I most commonly see in the Astro docs, where variable names are mostly translated as far as I've seen, not because of a personal choice. It's even stated in the README of the pt-br translation that everything that is not part of an existing API should be translated. Is there such an agreement in the Starlight translations? 🤔

Reply:
I searched a bit in the current translation files and there seems to be the both cases happening, with and without translation, so I don't know what would be the best way to proceed.

Reply:
I personally wouldn't mind keeping them in English, as its the norm I'm used to, but the Astro docs README has a good point:

> Comments, example components, variables, file paths and so on should be translated if they are not part of the Astro API, JavaScript or other libraries. This is intended to make the example codes as unrestrictive as possible for readers with lower English proficiency, bearing in mind that a fundamental part of understanding some pages is seeing the example codes and the comments that come with them.

Reply:
It's true that we have way less code examples involving variables in the Starlight Docs compared to the Astro Docs and I think in the Starlight repo, we are close to the guidelines from https://contribute.docs.astro.build/guides/i18n/#code-samples

> each language is free to decide whether or not they want to translate titles, variables, string values, function names, etc.

So I'll let you decide if you prefer to keep them as-is or revert them back to the original English version.

Let me know what you think and I'll either wait for an update or merge as-is.

Reply:
Additional question: would you be able to link the document where you found your quote? I checked https://github.com/withastro/docs/blob/main/README.md but I don't think it's in there 🤔

Reply:
I personally would prefer to keep code in English, but that's because I'm used to it. Considering the Astro docs pt-br translation reasoning of making things more accessible even for people with low English proficiency, and that I'm all pro accessible content, I thinks its better to translate the variables because it won't hurt anyone.

Reply:
> Additional question: would you be able to link the document where you found your quote? I checked [withastro/docs@`main`/README.md](https://github.com/withastro/docs/blob/main/README.md?rgh-link-date=2024-05-02T17%3A14%3A53Z) but I don't think it's in there 🤔

Sorry, forgot to do that. Here it is: https://github.com/withastro/docs/blob/main/src/i18n/pt-br/README.md#c%C3%B3digo

Reply:
Ah, it's the language specific README, I see. Makes total sense, thanks a lot for discussing this through.

I'll take one last quick look at the changes but I think this should be ready to go.